### PR TITLE
Minor improvements on Angular snippets

### PR DESCRIPTION
**Pull request by pBouillon**
Hey there !

I reached you out on Discord about some issues that I found on the Angular templates
I took the liberty to fix the other I noticed in the other Angular snippets, I also added a snippet for the `Slot Fallback` section.

My changes are:

- Formatting (one liners in `.html` has been moved to `template: '...'` 
- Improvements (removed unnecessary Angular code when there was some, like setting the component change strategy)
- Missing snippets (the `Slot Fallback` one)

Please feel free to tell me if you want me to update anything else on the changes I made !

### CODE REVIEW

File: content/4-component-composition/4-slot-fallback/angular/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+	selector: 'app-root',
+	template: '<app-funny-button>Click me !</app-funny-button>',
```
Comment (matschik) [inline content/4-component-composition/4-slot-fallback/angular/app.component.ts:5]: In slot fallback, we want to show the component using the fallback, and the same component not using the fallback: https://component-party.dev/#slot-fallback
The component using the fallback is missing

File: content/6-form-input/1-input-text/angular/input-hello.component.ts [hunk 2]
```diff
@@ -1,17 +1,9 @@
 import { Component } from '@angular/core';
 
-interface InputEvent {
-	target: HTMLInputElement;
-}
-
 @Component({
 	selector: 'app-input-hello',
-	template: ` <input [value]="inputValue" (input)="onInput($event)" /> `,
+	template: '<input [(ngModel)]="inputValue" />',
```
Comment (matschik) [inline content/6-form-input/1-input-text/angular/input-hello.component.ts:5]: So much better ! 😍

**Comment by matschik** [comment]
Amazing ! You saw so many improvements !
Good thing about one-liners, the purpose was to show that you can declare HTML markup in a separate file.

File: content/4-component-composition/4-slot-fallback/angular/app.component.ts (hunk 1 above)
Comment (pBouillon) [reply content/4-component-composition/4-slot-fallback/angular/app.component.ts:5]: Nice catch, indeed I missed that one, I'm fixing it right away !

**Comment by pBouillon** [comment]
Understood, I let the templates containing more than one line into their dedicated `.html` files and also fixed the Angular example for the content projection fallback to match the other ones

Let me know if anything else need an update !

**Comment by matschik** [comment]
Just perfect 👌 
Thank you !

**Comment by nartc** [comment]
@pBouillon cc @matschik the Slot Fallback example is not correct. 

```html
<button #container>
  <ng-content></ng-content>
</button>

<button *ngIf="!container.hasChildNodes()">
  <span>No content found</span>
</button>
```

The first button would still show up as an empty button

**Comment by pBouillon** [comment]
Indeed it does, I missed that one, sorry about that

[@nartc's suggestion](https://github.com/matschik/component-party/issues/29#issuecomment-1117343154) would surely be better
Would you like to create the new PR with your code? Otherwise I can create it if you prefer!

**Comment by nartc** [comment]
@pBouillon I can send a PR later today. If you're available and want to PR, don't let me stop you :D

**Comment by pBouillon** [comment]
@nartc Had some time to spare, I just submitted it, hope I get it right this time 🤞

**Pull request merged**